procedures/unit-testing-basics.ipf

```diff
@@ -54,6 +54,19 @@ static Function ClearRTError()
 	variable err = GetRTError(1)
 End
 
+/// @brief Check wether the function reference points to
+/// the prototype function or to an assigned function
+///
+/// Due to Igor Pro limitations you need to pass the function
+/// info from `FuncRefInfo` and not the function reference itself.
+///
+/// @return 1 if pointing to prototype function, 0 otherwise
```

Comment:
Please change to:

/// @return 0 if pointing to prototype function, 1 otherwise

Reply:
@MichaelHuth Thanks fixed. I also renamed the function to not clash with the same function in MIES.